common/display/displayplanemanager.cpp

```diff
@@ -188,7 +188,10 @@ bool DisplayPlaneManager::ValidateLayers(
 
       last_plane.AddLayer(i->GetZorder(), i->GetDisplayFrame(), false);
     }
-
+    if (!last_plane.GetOffScreenTarget() || last_plane.GetSurfaces().size() < 3) {
+    SetOffScreenPlaneTarget(last_plane);
```

Comment:
Below we already do           if (fall_back) {
            ResetPlaneTarget(last_plane, commit_planes.back());
            validate_final_layers = true;
          }

perhaps last_plane.SwapSurfaceIfNeeded(); should be called after thatt ?